File: remove_bg.py
from PIL import Image

def remove_white_bg(image_path):
    img = Image.open(image_path)
    img = img.convert("RGBA")
    datas = img.getdata()

    newData = []
    # Tolerance for white
    for item in datas:
        # if pixel is mostly white (e.g. RGB all > 220)
        if item[0] > 220 and item[1] > 220 and item[2] > 220:
            newData.append((255, 255, 255, 0)) # transparent
        else:
            newData.append(item)

    img.putdata(newData)
    img.save(image_path, "PNG")

remove_white_bg('src/assets/logo.png')
print("Background removed successfully")

File: remove_bg.js
import { Jimp } from "jimp";

async function main() {
    try {
        const image = await Jimp.read('src/assets/logo.png');
        
        image.scan(0, 0, image.bitmap.width, image.bitmap.height, function(x, y, idx) {
            const r = this.bitmap.data[idx + 0];
            const g = this.bitmap.data[idx + 1];
            const b = this.bitmap.data[idx + 2];
            
            // If the pixel is very light/white (e.g., r,g,b > 240)
            if (r > 240 && g > 240 && b > 240) {
                // Set alpha to 0 (transparent)
                this.bitmap.data[idx + 3] = 0;
            }
        });

        await image.write('src/assets/logo.png');
        console.log("Background removed successfully!");
    } catch (error) {
        console.error("Error processing image:", error);
    }
}

main();
